### PR TITLE
Prepare release r1.3

### PR DESCRIPTION
**Pull request by akoshunyadi**
#### What type of PR is this?

* subproject management

#### What this PR does / why we need it:

Prepare patch release r1.3
- update the modified OAS files (compared to r1.2)
- update change log

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +18,89 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent alpha or release-candidate API versions, the delta with respect to the previous pre-release
 * for a public API version, the consolidated changes since the release of the previous public API version
 
+# r1.3
+## Release Notes
+
+This release contains the definition and documentation of
+* device-roaming-status v0.6.0
```
Comment (sachinvodafone) [inline CHANGELOG.md:25]: Just checking, will it be on 0.6.0 where others on 0.6.1

Comment (akoshunyadi) [reply CHANGELOG.md:25]: There wasn't any change since r1.2. The APIs are independent, so the numbering will run separately.

**Review by bigludo7 — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,6 +18,89 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent alpha or release-candidate API versions, the delta with respect to the previous pre-release
 * for a public API version, the consolidated changes since the release of the previous public API version
 
+# r1.3
+## Release Notes
+
+This release contains the definition and documentation of
```
Comment (hdamker) [inline CHANGELOG.md:24]:
```suggestion
This patch release contains the definition and documentation of
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,6 +18,89 @@ The below sections record the changes for each API version in each release as fo
...
+* device-roaming-status v0.6.0
+* device-roaming-status-subscriptions v0.6.1
+* device-reachability-status v0.6.1
+* device-reachability-status-subscriptions v0.6.1
+
+The API definition(s) are based on
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.0
+
+Note: the previous device-status API with roaming and connectivity endpoints has been split into 4 specific APIs
```
Comment (hdamker) [inline CHANGELOG.md:34]:
```suggestion
Note: these patch release notes are listing only the change compared to the [r1.2](https://github.com/camaraproject/DeviceStatus/releases/tag/r1.2) release. For the full list of changes of the 0.6.0 API versions compared to 0.5.x see [r1.2](#r12).
```

**Review by hdamker — DISMISSED** [review]
@akoshunyadi please touch also shortly the API-Readiness-Checklist (first line only). And add @camaraproject/release-management_maintainers - all LGTM but as it is a public review and there need to be some updates done (e.g. the CAMARA Readme), Release Management must be involved.

**Comment by akoshunyadi** [comment]
Review findings done, the review by https://github.com/orgs/camaraproject/teams/release-management_maintainers was already requested

**Review by sachinvodafone — APPROVED** [review]

**Comment by akoshunyadi** [comment]
@camaraproject/release-management_maintainers kindly asking for final review

File: code/API_definitions/device-reachability-status-subscriptions.yaml
Comment (tanjadegroot) [inline code/API_definitions/device-reachability-status-subscriptions.yaml:1]: - line 5: change "customer" to "API consumer"
- line 17: the Device identifier "Network Access Identifier" is not allowed in Fall24 release, so a not should be added saying it is not supported for this API version.
- line 62: info.description field: update the auth section to the latest text from the ICM CAMARA-API-access-and-user-consent.md document (removing link with "main").

File: code/API_definitions/device-reachability-status.yaml
Comment (tanjadegroot) [inline code/API_definitions/device-reachability-status.yaml:1]: - line 5: change "customer" to "API consumer"
- line 16: the Device identifier "Network Access Identifier" is not allowed in Fall24 release, so a not should be added saying it is not supported for this API version.
- line 33: info.description field: update the auth section to the latest text from the ICM CAMARA-API-access-and-user-consent.md document. (removing link with "main")

File: code/API_definitions/device-roaming-status.yaml
Comment (tanjadegroot) [inline code/API_definitions/device-roaming-status.yaml:1]: - line 5: change "customer" to "API consumer"
- line 24: the Device identifier "Network Access Identifier" is not allowed in Fall24 release, so a not should be added saying it is not supported for this API version.
- line 40: info.description field: update the auth section to the latest text from the ICM CAMARA-API-access-and-user-consent.md document (removing link with "main").

File: documentation/API_documentation/device-reachability-status-API-Readiness-Checklist.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # API Readiness Checklist
 
-Checklist for device-reachability-status 0.6.0 in r1.2.
+Checklist for device-reachability-status 0.6.1 in r1.3.
```
Comment (tanjadegroot) [inline documentation/API_documentation/device-reachability-status-API-Readiness-Checklist.md:3]: 0.6. 1 - remove space -> 0.6.1
r1. 3 - remove space -> r1.3

File: documentation/API_documentation/device-reachability-status-subscriptions-API-Readiness-Checklist.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # API Readiness Checklist
 
-Checklist for device-reachability-status-subscriptions 0.6.0 in r1.2.
+Checklist for device-reachability-status-subscriptions 0.6.1 in r1.3.
```
Comment (tanjadegroot) [inline documentation/API_documentation/device-reachability-status-subscriptions-API-Readiness-Checklist.md:3]: remove spaces in numbering schemes

File: documentation/API_documentation/device-roaming-status-subscriptions-API-Readiness-Checklist.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # API Readiness Checklist
 
-Checklist for device-roaming-status-subscriptions 0.6.0 in r1.2.
+Checklist for device-roaming-status-subscriptions 0.6.1 in r1.3.
```
Comment (tanjadegroot) [inline documentation/API_documentation/device-roaming-status-subscriptions-API-Readiness-Checklist.md:3]: remove spaces in numbering schemes

File: CHANGELOG.md [hunk 7]
```diff
@@ -17,6 +18,89 @@ The below sections record the changes for each API version in each release as fo
...
+
+This patch release contains the definition and documentation of
+* device-roaming-status v0.6.0
+* device-roaming-status-subscriptions v0.6.1
+* device-reachability-status v0.6.1
+* device-reachability-status-subscriptions v0.6.1
+
+The API definition(s) are based on
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.0
```
Comment (tanjadegroot) [inline CHANGELOG.md:32]: you could update the ICM version to 0.2.1 (there is no API impact)

**Comment by tanjadegroot** [comment]
In addition to the PR comments, I saw some further points:

README.me:
- line 10: "API family" --> "APIs"
- line 14: "customer" --> "API consumer"
- line 15: "has lost connectivity" --> "is not connected"

File: documentation/API_documentation/device-reachability-status-API-Readiness-Checklist.md (hunk 4 above)
Comment (akoshunyadi) [reply documentation/API_documentation/device-reachability-status-API-Readiness-Checklist.md:3]: @tanjadegroot there's no space actually

File: documentation/API_documentation/device-reachability-status-subscriptions-API-Readiness-Checklist.md (hunk 5 above)
Comment (akoshunyadi) [reply documentation/API_documentation/device-reachability-status-subscriptions-API-Readiness-Checklist.md:3]: @tanjadegroot there are no spaces

File: documentation/API_documentation/device-roaming-status-subscriptions-API-Readiness-Checklist.md (hunk 6 above)
Comment (akoshunyadi) [reply documentation/API_documentation/device-roaming-status-subscriptions-API-Readiness-Checklist.md:3]: @tanjadegroot there are no spaces

File: CHANGELOG.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (akoshunyadi) [reply CHANGELOG.md]: version changed

Comment (akoshunyadi) [reply CHANGELOG.md]: version changed

File: code/API_definitions/device-roaming-status-subscriptions.yaml
[the remark this answers is not in the document]
Comment (akoshunyadi) [reply code/API_definitions/device-roaming-status-subscriptions.yaml:1]: We will update the documentation for the next main release. See #222

**Review by bigludo7 — DISMISSED** [review]
LGTM

**Comment by akoshunyadi** [comment]
@bigludo7 now it should be perfect 🙂

**Review by bigludo7 — APPROVED** [review]
LGTM :)

**Comment by tanjadegroot** [comment]
Great ! OK from Release Management /LGTM

**Pull request merged**